src/ripple/test/jtx/JTx.h

```diff
@@ -56,12 +59,99 @@ struct JTx
     {
     }
 
+#if defined(_MSC_VER) && _MSC_VER <= 1800
```

Comment:
Fair enough. I tend to price moves very cheaply - on the order of assigning a pointer or maybe a couple pointers. I'm certainly fine keeping this as two functions.